doc/apple2.sgml

```diff
@@ -427,8 +427,12 @@ The names in the parentheses denote the symbols to be used for static linking of
 <descrip>
 
   <tag><tt/a2.ssc.ser (a2_ssc_ser)/</tag>
-  Driver for the Apple&nbsp;II Super Serial Card. Supports up to 19200 baud,
-  requires hardware flow control (RTS/CTS) and does interrupt driven receives.
+  Driver for the Apple&nbsp;II Super Serial Card.
+  They are extension cards for the II, II+, IIe, and the Apple //c and //c+ have
+  the same hardware and firmware integrated.
```

Comment:
I'm no native speaker but it seems to me that this should rather be singular in order to fit to the sentence above and below.